test/extensions/access_loggers/file/BUILD

```diff
@@ -14,9 +14,13 @@ envoy_package()
 envoy_extension_cc_test(
     name = "config_test",
     srcs = ["config_test.cc"],
-    extension_names = ["envoy.access_loggers.file"],
+    extension_names = [
+        "envoy.access_loggers.file",
+        "envoy.formatter.metadata",
+    ],
     deps = [
         "//source/extensions/access_loggers/file:config",
+        "//source/extensions/formatter/metadata:config",
```

Comment:
Please don't do this. Basically, we will use the `extension_build_config.bzl` to control whether an extension should be linked or not in the final binary.

This change introduce unnecessary dependency and break the above rule.